doc/example-spouse.md

```diff
@@ -88,12 +88,17 @@ Then we compile our application, as we must do whenever we change `app.ddlog`:
 deepdive compile
 ```
 
-Finally, we tell DeepDive to execute the steps to load the `articles` table:
+Finally, we tell DeepDive to execute the steps to load the `articles` table using the `input/articles.tsv.sh` script. You must have the [full corpus](http://research.signalmedia.co/newsir16/signal-dataset.html) downloaded.
 
 ```bash
 deepdive do articles
 ```
 
+Alternatively, a sample of 1000 articles can be loaded directly by issuing the following command:
+```bash
+deepdive load articles input/articles-1000.tsv.bz2
```

Comment:
I forgot `deepdive do` was actually doing much more and you need these commands to run before `deepdive load`:

```bash
deepdive do init/app
deepdive create table articles
deepdive load articles input/articles-1000.tsv.bz2
deepdive mark done articles
```

We could come up with a new shorthand for this kind of side-loading operation or revise existing `deepdive load` or `deepdive do`.